src/client.c

```diff
@@ -547,6 +550,12 @@ int main(int argc, char *argv[])
                 while (clock() - begin < TIMEOUT);
             }
 
+            render_board(gcb);
+            render_tiles(gcb, gcb->turn);
+            render_score_board();
+            render_score(rcb, !gcb->turn);
+            refresh();
```

Comment:
I think we could move this `refresh()` before `goto END_OF_GAME` or right after `END_OF_GAME:` so that it won't refresh twice every time

Reply:
I was originally thinking the same issue. I'll fix this with your advice.